agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryChannel.java

```diff
@@ -86,6 +85,10 @@
   //  operationLogger?
   private final AtomicBoolean friendlyExceptionThrown = new AtomicBoolean();
 
+  private static final int MAX_STATSBEAT_ERROR_COUNT = 10;
```

Comment:
10 seems like a lot, I think if the first three requests all fail, that's a pretty solid indicator

Reply:
i gave an example responding to your other comments above. i think 10 is better and it's consistent with breeze retry limit.

Reply:
10 is breeze redirect limit, not retry limit?

Reply:
https://github.com/microsoft/Telemetry-Collection-Spec/pull/51/files#:~:text=10%20redirects%20triggered%20in%20succession%20(circular%20redirect)%2C%20we%20will%20drop%20the%20data%20and%20count%20it%20as%20an%20exception.

redirect, not retry.

Reply:
right, so what does that have to do with why we should use 10 here?